code/renderers/svelte/src/__test__/Button.svelte

```diff
@@ -1,8 +1,13 @@
 <script lang="ts">
-  import { createEventDispatcher } from 'svelte';
+  const { createEventDispatcher } = require('svelte');
```

Comment:
This seems strange, I thought svelte was all-in on ESM.  Why was this change needed?

Reply:
This is what I did previously to get those tests passing: https://github.com/storybookjs/storybook/pull/24415/commits/ca0394d77c0d41009181e73848634cdd6a3bd3c8

I see now there's an `// eslint-disable-next-line import/no-unresolved` comment above the import, so maybe that has something to do with it?  Why wouldn't that import be resolved?  And if it's not, it would explain why the files can't be parsed.

Reply:
Seems like this was not needed after all, it was causing issues which got solved after undoing these changes